queries.sql

```diff
@@ -6,3 +6,29 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Vet clinic database: query and update animals table
+BEGIN;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE name LIKE IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(name) FROM animals;
+SELECT COUNT(name) FROM animals WHERE escape_attempts=0;

```

Comment:
- [ ] Great job on your queries! 😊 However, there's a small issue that needs attention. 

  - Based on the data you inserted into your database, it appears that the number of animals who have never tried to escape is 2. This conclusion doesn't align with the provided screenshot in the pull request summary, which indicates 3. To ensure accuracy, please review the query and make the necessary adjustments to return the correct result, which should be 2 in this case. 

Additionally, don't forget to update the screenshot in the pull request summary with the revised result to maintain consistency and clarity in our documentation.

Happy coding! 😊🚀🍀

Reply:
Hi, @Strangeal,
thank you for your review. I have made the necessary corrections. However, I want to inform you that based on the data i inserted into the database, the number of animals who have never tried to escape is actually 3. Kindly review the data again. 
Thanks.